### PR TITLE
Fix: remove cartography part 7

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Remove Cartography from view controllers

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Components/Giphy/GiphyConfirmationViewController.swift [hunk 1]
```diff
@@ -57,7 +56,7 @@ final class GiphyConfirmationViewController: UIViewController {
 
         let closeImage = StyleKitIcon.cross.makeImage(size: .tiny, color: .black)
         navigationItem.rightBarButtonItem = UIBarButtonItem(image: closeImage, style: .plain, target: self, action: #selector
-            (GiphySearchViewController.onDismiss))
+                                                                (GiphySearchViewController.onDismiss))
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Components/Giphy/GiphyConfirmationViewController.swift:59]: Do we need this white space?

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Components/Giphy/GiphyConfirmationViewController.swift]: fixed the line wrap. thx

**Review by billypchan — COMMENTED** [review]
fix line break

File: Wire-iOS/Sources/UserInterface/Components/Giphy/GiphyConfirmationViewController.swift [hunk 2]
```diff
@@ -56,8 +55,10 @@ final class GiphyConfirmationViewController: UIViewController {
         }
 
         let closeImage = StyleKitIcon.cross.makeImage(size: .tiny, color: .black)
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: closeImage, style: .plain, target: self, action: #selector
-            (GiphySearchViewController.onDismiss))
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: closeImage, 
+                                                            style: .plain, 
+                                                            target: self, 
+                                                            action: #selector(GiphySearchViewController.onDismiss))
```
[the remark this answers is not in the document]
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Components/Giphy/GiphyConfirmationViewController.swift:61]: fixed again

**Review by KaterinaWire — APPROVED** [review]

**Review by agisilaos — APPROVED** [review]
🚀

**Pull request merged**